gcp/modules/gcp-stackdriver-exclusion/main.tf

```diff
@@ -0,0 +1,29 @@
+terraform {
+  backend "gcs" {}
+}
+
+variable "project_id" {}
+variable "serviceaccount_key" {}
+variable "exclusions" {
+  default = {
+    # Calico logs don't always set 'severity' correctly, so instead we match on
+    # 'textPayload'.
+    "calico-info" = "resource.type=container AND logName = (projects/__var.project_id__/logs/calico-node OR projects/__var.project_id__/logs/calico-typha) AND textPayload:\"[INFO]\""
```

Comment:
Since we have the `project_id` variable available here, we might be able to go without the replacement bit?